b2sdk/account_info/sqlite_account_info.py

```diff
@@ -320,9 +320,9 @@ def get_account_id(self):
         """
         return self._get_account_info_or_raise('account_id')
 
-    def get_account_id_or_app_key_id(self):
+    def get_application_key_id(self):
         """
-        Return account ID or appliction key ID.
+        Return application key ID or Account ID.
```

Comment:
```suggestion
        Return application key id.
```
How `application key id` and `account id` relate to each other, why they can be equal and when, is outside of scope of this docstring. Whoever reads this, should already know what application key id is, or should go check it in some other docs.

Reply:
Are you suggesting that I leave the original docstring or that its fine?

Reply:
The old version was:
```
Return account ID or appliction key ID.
```
Your change is:
```
Return application key ID or Account ID.
```
and I suggest to:
```
Return application key id.
```
(if you want to accept my suggestion, you can click a button on github)

Reply:
I believe after our convention discussion in slack it will be 
`Return application key ID.`